README.md

```diff
@@ -1 +1,77 @@
-# covid-19_tracking_app
\ No newline at end of file
+# Covid-19_Tracker_App
+This Mobile web applicat is a Covid-19 data tracking App by Country , the data is from [Narrativa API](https://covid19tracking.narrativa.com/index_en.html).
+
+![screenshot](./React-App-1.png)
+![screenshot](./Reacr-app.png)
+
+## Built With
+
+- React
+- JSX
+- Redux
+- [CORONAVIRUS DATA API - COVID-19 TRACKING PROJECT](https://covid19tracking.narrativa.com/index_en.html)
+
+## Live Demo
+
+- N/A
```

Comment:
- [ ] You should add a live demo of your capstone, you can use Heroku, Netlify or GitHub Pages